vault/auth_mount.go

```diff
@@ -95,6 +95,16 @@ func authMountTune(client *api.Client, path string, configured interface{}) erro
 	return nil
 }
 
+func authMountTuneGet(client *api.Client, path string) (map[string]interface{}, error) {
+	tune, err := client.Sys().MountConfig(path)
+	if err != nil {
+		log.Printf("[ERROR] Error when reading tune config from path '%q/tune': %s", path, err)
```

Comment:
`%q` is wrapping `path` in quotes here? So it'll come out looking like `path '"auth/jwt"/tune'`? I wonder if something like this would be clearer:

```suggestion
		log.Printf("[ERROR] Error when reading tune config from path %q: %s", path+"/tune", err)
```